server/test/com/cloud/user/AccountManagerImplTest.java

```diff
@@ -231,6 +253,73 @@ public void cleanup() {
         CallContext.unregister();
     }
 
+    public UsageEventUtils setupUsageUtils() {
+        _usageEventDao = new MockUsageEventDao();
+        UsageEventUtils utils = new UsageEventUtils();
+
+        Map<String, String> usageUtilsFields = new HashMap<String, String>();
+        usageUtilsFields.put("usageEventDao", "_usageEventDao");
+        usageUtilsFields.put("accountDao", "_accountDao");
+        usageUtilsFields.put("dcDao", "_dcDao");
+        usageUtilsFields.put("configDao", "_configDao");
```

Comment:
@ProjectMoon I got lost thinking it was in the main code base re-allocating a set of static values on every invocation would cause GC churn.  Not a significant concern in a test class.